python/cugraph/tests/utils.py

```diff
@@ -14,6 +14,51 @@
 import cudf
 import pandas as pd
 
+#
+# Datasets are numbered based on the number of elements in the array
+#
+DATASETS_1 = ['../datasets/netscience.csv']
```

Comment:
We could also remove datasets that don't bring much value? Not sure if dolphins/polbooks are really testing something different than karate.

Lots of duplicates in the lists also. I wonder if we could just group them by directed/undirected sets.

Reply:
You hit on the point/scope of this refactoring.  The first step was to just centralize the data.  The next is to make better tests.